src/debugger/com.c

```diff
@@ -819,6 +822,9 @@ void xdebug_debug_init_if_requested_at_startup(void)
 		}
 		xdebug_init_debugger();
 	}
+	if (XDEBUG_MODE_IS_ONLY_DEBUG() && XG_DBG(remote_connection_enabled) == 0) {
```

Comment:
When we do the initial connection attempt, if we are only using the debug mode and we have not connected to any client (either because the debugger was not started or because no client is available) then we disable the debugger